CHANGELOG.md

```diff
@@ -3,6 +3,24 @@
 ## v0.5.0 (WIP)
 
   * [components] ErrorTag: Renamed prop `phx_trigger_action` to `trigger_action`
```

Comment:
This should also go to breaking changes, right?

And maybe remove the `[components]` part?  Seems out of place with the rest of the changelog.